docs/editor_guide.md

```diff
@@ -94,8 +93,69 @@ these steps need to be taken:
    ```
 2. set the GitHub topics, if they haven't been set yet. Topics should include at least `lesson` and the appropriate life-cycle badge.
 
-### After acceptance
+### Performing editorial checks
+Using the checklist above, perform the basic editorial checks to ensure that the
+lesson is ready for review.
+Share the results of the checklist with the authors in the thread,
```

Comment:
does the "thread" here refer to the issue in this repo that represents the submission?

Reply:
Yes, most communication should take place within the issue thread for the review in this repository.